Spigot-Server-Patches/0574-Lazily-track-plugin-scoreboards-by-default.patch

```diff
@@ -39,7 +39,20 @@ index 6ea491f6308317059c4bc6735abbdce370df0f34..88bc3fbfa3bfb27cc4f9003a99cbce6e
  
      CraftScoreboard(Scoreboard board) {
          this.board = board;
-@@ -66,6 +67,12 @@ public final class CraftScoreboard implements org.bukkit.scoreboard.Scoreboard {
+@@ -45,6 +46,12 @@ public final class CraftScoreboard implements org.bukkit.scoreboard.Scoreboard {
+         Validate.isTrue(name.length() <= 16, "The name '" + name + "' is longer than the limit of 16 characters");
+         Validate.isTrue(board.getObjective(name) == null, "An objective of name '" + name + "' already exists");
+         CraftCriteria craftCriteria = CraftCriteria.getFromBukkit(criteria);
++        // Paper start
```

Comment:
Add why this is added here

Reply:
Done

Reply:
I more meant an extra comment pointing out how the block comment you showed internally broke this 😅

Reply:
Ok, should be there now